script.module.t1mlib/addon.xml

```diff
@@ -13,7 +13,7 @@
         <email></email>
         <website></website>
         <forum>http://forum.kodi.tv/showthread.php?tid=247163</forum>
-        <source>https://github.com/learningit/repository.t1m.v2.addons/tree/master/src/script.module.t1mlib</source>
+        <source>https://github.com/xbmc/repo-scripts/tree/isengard/script.module.t1mlib</source>
```

Comment:
Well if this was valid/wanted we could generate this data for our devs. 
I think the idea behind it was too drive traffic to your repo with possible questions, PRs or reviews. Which now might end up on our repos instead. Our they realize it and will have to search for the correct repo to do a PR against.